### PR TITLE
reorganize and delete a lot of bloat

### PR DESCRIPTION
**Pull request by Rexicon226**

### CODE REVIEW

**Comment by Mineinjava** [comment]
works on my machine

File: DNoise/helper_functions.py
Comment (Mineinjava) [inline DNoise/helper_functions.py:1]: YAY!

File: Terrain/border.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import copy
 
 import Terrain.terraingen as terraingen
-from Terrain.timers import FunctionTimer
+from Terrain.timer import FunctionTimer
```
Comment (Mineinjava) [inline Terrain/border.py:4]: timing is good

File: Terrain/timer.py
Comment (Mineinjava) [inline Terrain/timer.py:1]: looks good

File: Terrain/visualizer.py
Comment (Mineinjava) [inline Terrain/visualizer.py:1]: i don't remember ever writing most of this

**Review by Mineinjava — APPROVED** [review]
LGTM i guess

File: Terrain/visualizer.py
Comment (Rexicon226) [reply Terrain/visualizer.py:1]: neither do i

File: Terrain/border.py (hunk 1 above)
Comment (Rexicon226) [reply Terrain/border.py:4]: timing is a good name yes, i will change

**Pull request merged**